.rubocop.yml

```diff
@@ -28,9 +28,17 @@ Style/Documentation:
     - 'spec/**/*'
     - 'app/**/*'
 
+Style/ConditionalAssignment:
+  Exclude:
+    - 'app/helpers/*'
+    
 Style/Next:
   Enabled: true
 
+Style/NegatedIf:
+  Exclude:
+    - 'app/helpers/*'
```

Comment:
If the negated condition noted by Hound is dropped (turned into an else or early return/guard clause) will this still be needed?